app/scripts/modules/netflix/instance/titus/netflixTitusInstanceDetails.controller.js

```diff
@@ -47,7 +48,7 @@ module.exports = angular.module('spinnaker.netflix.instance.titus.controller', [
     this.getBastionAddressForAccount = (account, region) => {
       return accountService.getAccountDetails(account).then((details) => {
         this.bastionHost = details.bastionHost || 'unknown';
-        this.apiEndpoint = _.where(details.regions, {name: region})[0].endpoint;
+        this.apiEndpoint = _.filter(details.regions, {name: region})[0].endpoint;
```

Comment:
@anotherchrisberry Yea, there's a bunch of places where we can drop lodash and use es6 API functions but I thought it better to do that in a separate PR.  Thoughts?